cdap-ui/app/cdap/services/helpers.js

```diff
@@ -264,6 +264,8 @@ const isBatchPipeline = (pipelineType) => {
   return ['cdap-data-pipeline'].indexOf(pipelineType) !== -1;
 };
 
+const NUMBER_TYPES = ['integer', 'short', 'long', 'float', 'double'];
```

Comment:
nit: When you rebase you'll see a new file called `global-constants.js` inside this same directory (which was moved over from Angular side). Can you move this constant there? This file seems to be used more for helper functions